yt_dlp/extractor/lrt.py

```diff
@@ -118,7 +117,7 @@ class LRTRadioIE(LRTBaseIE):
         'url': 'https://www.lrt.lt/radioteka/irasas/2000359728/nemarios-eiles-apie-pragarus-ir-skaistyklas-su-aiste-kiltinaviciute',
         'info_dict': {
             'id': '2000359728',
-            'ext': 'm4a',
+            'ext': 'mp4',
```

Comment:
Ah, I forgot the site is mostly(/all?) audio. 

Maybe we should remove the `'mp4'` arg from the `_extract_m3u8_formats` call in that case.

However, audio-only AAC-over-HLS can be problematic sometimes when we download it as `m4a`.

There are a handful of sites/extractors where if we are downloading AAC HLS as `m4a`, then we need to force a remux (via an ugly `'container': 'm4a_dash'` hack).

So it might actually desirable to force the `mp4` extension here, since `mp4` HLS downloads are automatically remuxed by yt-dlp after download.

Would you be up for testing a couple full downloads and seeing how they playback / if they are scrubbable in the media player / etc?

Reply:
Sure! I'll test it and will post an update soon.

Reply:
Tested both, mp4 and m4a on a few random episodes. No issues with both. Should we leave mp4 in that case?

Reply:
Thanks. Nah, let's remove it then.

Reply:
```suggestion
            'ext': 'm4a',
```